src/foam/dao/history/HistoryRecord.js

```diff
@@ -50,6 +50,12 @@ foam.CLASS({
       name: 'updates',
       label: 'Updated Properties',
       documentation: 'Properties updated, contains new and old values.'
+    },
+    {
```

Comment:
can you move this up under 'user', so natural ordering in detail view will place them together.

Reply:
sure, will do